app/src/sharedTest/java/org/oppia/app/settings/profile/ProfileRenameActivityTest.kt

```diff
@@ -75,6 +77,17 @@ class ProfileRenameActivityTest {
     }
   }
 
+  @Test
+  fun testProfileRenameActivity_inputNewName_clickSave_configurationChange_checkNameIsSaved() {
```

Comment:
This test case is not very helpful. Instead we should write something link this:

`testProfileRenameActivity_configurationChange_checkSaveIsDisabled`
`testProfileRenameActivity_inputNewName_configurationChange_checkSaveIsEnabled`
`testProfileRenameActivity_inputNewName_configurationChange_inputTextExists`

Mostly the last test case will fail and in that case you will need the concept of `onSavedInstance` to save the the string typed in edittext before orientation change and show that again in edittext after orientation change.

Reply:
done